projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -960,7 +960,9 @@ public function maybe_set_first_published_status( $new_status, $old_status, $pos
 	 * @return bool
 	 */
 	public function first_published_status_meta_auth_callback() {
-		if ( current_user_can( 'publish_posts' ) ) {
+		$capability = apply_filters( 'jetpack_subscriber_was_published_capability', 'publish_posts' );
```

Comment:
I'd suggest adding a docblock, so our Codex can be automatically updated with the new filter on the next release. You can check other filters in that file for the expected format:
https://github.com/Automattic/jetpack/blob/b77b183164ca8f0d71a0bd0bb8fb4079c7c3dec0/projects/plugins/jetpack/modules/subscriptions.php#L173-L181

You can use `@since $$next-version$$` as the since version number; it will be automatically replaced on release day.